python/lsst/analysis/drp/colorColorFitPlot.py

```diff
@@ -305,10 +330,11 @@ def colorColorFitPlot(self, catPlot, plotInfo, fitParams):
                 SNsUsed = (catPlot[SNBand + "_" + plotInfo["SNFlux"]].values[fitPoints]
```

Comment:
Is it possible for SNsUsed to be empty? I imagine the fit likely would have failed in that case, but if there's a chance it would keep going and result in an infinite loop again, maybe an explicit check would be worthwhile.

Reply:
Another good idea.  I put in a check on `len(fitParams["fitPoints"])` above with:
```
if len(fitParams["fitPoints"]) < 1:
    raise ValueError("No fitPoints for {}".format(dataId))
```